sidebars.js

```diff
@@ -418,6 +419,22 @@ module.exports = {
         'cli-reference',
         'semgrep-appsec-platform/json-and-sarif'
       ]
+    },
+    {
+      type: 'category',
```

Comment:
I don't think we should overload this header; I think this is the only spot where the green header acts as a link, which is confusing. I don't know what the alternative is -- maybe follow the convention of the otherlinks?